manifests/0000_50_cluster-monitoring-operator_02-role.yaml

```diff
@@ -30,31 +30,77 @@
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
+  creationTimestamp: null
```

Comment:
This is the only addition to the generated manifest. The other changes are only positional ones.

This addition is not intended but I can't get rid of it for some reason.

Reply:
We seem to have that elsewhere, so it's okay. https://github.com/openshift/cluster-monitoring-operator/blob/d32a7b0d759d28221b4a4a3f5429c8b00738790a/manifests/0000_50_cluster-monitoring-operator_00_0servicemonitor-custom-resource-definition.yaml#L6